velox/exec/tests/JoinFuzzer.cpp

```diff
@@ -887,6 +1074,25 @@ void JoinFuzzer::go() {
   }
 }
 
+void JoinFuzzer::cleanup() {
+  keyTypes_.clear();
```

Comment:
It feels like it is easy to forget some field here.

Reply:
Yeah, put the test config in a separate subclass and reset all together.